Jenkinsfile

```diff
@@ -126,6 +126,14 @@ node('lisk-nano') {
           Xvfb :1$N -ac -screen 0 1280x1024x24 &
 
           # Run end-to-end tests
+
+          withCredentials([string(credentialsId: 'lisk-nano-testnet-passphrase', variable: 'TESTNET_PASSPHRASE')]) {
```

Comment:
You need to move this `withCredentials` statement to wrap the whole `sh` above, i.e. you need to have:
```[...]
withCredentials([...]) {
    sh ```
        [...]
        npm run --silent e2e-test:testnet:custom -- --params.baseURL file://$WORKSPACE/app/build/index.html --params.liskCoreURL http://127.0.0.1:400$N  --params.testnetPassphrase $TESTNET_PASSPHRASE --params.testnetCustomNode true
        [...]
    ```
}
[...]```